core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/WorkspaceConfig.java

```diff
@@ -57,4 +57,9 @@
    * least 1 default environment and may contain N environments.
    */
   Map<String, ? extends Environment> getEnvironments();
+
+  /**
+   * Returns workspace config attributes. Workspace attributes must not contain null keys or values.
```

Comment:
Pls rephrase 2nd sentence to be clear that is it talking about _config_ attributes